setup.cfg

```diff
@@ -2,7 +2,8 @@
 universal=1
 
 [codespell]
-skip = *.json,*.cpp,*.c,.riot,.tox,.mypy_cache,.git,*ddtrace/vendor
+skip = *.json,*.cpp,*.c,.riot,.tox,.mypy_cache,.git,*ddtrace/vendor,tests/contrib/openai/cassettes/*
+exclude-file = .codespellignorelines
```

Comment:
Do we need a codespell ignores?

Reply:
Yes, one of the tests specifically includes a typo on purpose which codespell flags as a typo.